pennylane/math/multi_dispatch.py

```diff
@@ -418,8 +419,8 @@ def get_trainable_indices(values, like=None):
     >>> def cost_fn(params):

```

Comment:
```suggestion
    >>> from pennylane import numpy as pnp
    >>> def cost_fn(params):
```
Maybe to make it explicit?